src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs

```diff
@@ -839,6 +851,18 @@ public ConnectorViewModel(WorkspaceViewModel workspace, ConnectorModel connector
             this.PropertyChanged += ConnectorViewModelPropertyChanged;
         }
 
+        private void HandleConnectorPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            switch (e.PropertyName)
+            {
+                case nameof(ConnectorModel.IsVisible):
+                    ConnectorModel connector = sender as ConnectorModel;
+                    IsVisible = connector.IsVisible;
```

Comment:
maybe check null just to prevent crash from event raised by 3rd party

Reply:
Done.